irohad/torii/command_service.cpp

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 #include "torii/command_service.hpp"
```

Comment:
If we'll use `"` symbol here, documents should be written at first, then all codes should follow the style (but currently, we use `<>` symbol for include path)